File: Aula 4 - Lista Linear Sequencial/aula_4.dart
List lista = [1, 2, 3, 4, 5, 6, 14, 47, 11, 9, 10];

buscaLista(List l, int busca) {
  int i = 0;
  print(l.length);
  while (i < l.length && l[i] != busca) {
    i++;
  }
  if (i != l.length) {
    return 'O resultado da pesquisa ${l[i]}, foi encontrado no índice ${i}';
  }
  return '';
}

void main() {
  print(buscaLista(lista, 10));
}

File: Aula 4 - Lista Linear Sequencial/aula_4.js
var lista = [1, 2, 3, 4, 5, 6];

function buscaLista(l, busca) {
    var i = 0;
    while (i < l.length && l[i] != busca) {
        i++
    }
    if (i != l.length) {
        console.log(`Resultado da pesquisa ${l[i]}, encontrado no índice ${i}`)
    }
}

buscaLista(lista, 7)